asserts/header_checks.go

```diff
@@ -33,6 +33,14 @@ func checkMandatory(headers map[string]string, name string) (string, error) {
 	if !ok {
 		return "", fmt.Errorf("%q header is mandatory", name)
 	}
+	return value, nil
+}
+
+func checkNotEmpty(headers map[string]string, name string) (string, error) {
+	value, err := checkMandatory(headers, name)
```

Comment:
Good catch. I changed the name as suggested, but left the error text using "mandatory".